### PR TITLE
Update documentation for new iteration entity access rules editor

### PR DESCRIPTION
**Pull request by brambrink**
Hi team,

In Studio Pro 10.17, we will introduce a new iteration of the entity access rules editor, based on customer feedback. We made quite a lot of changes compared to the old iteration. The documentation is updated to describe the new iteration. 

On of the changes of the new iteration is that we do not normalize the access rules anymore. I updated the warning to reflect this.

Moreover, we added a note that the beta entity access rules editor is scheduled to be GA (generally available) from 10.18.

### CODE REVIEW

File: content/en/docs/refguide/modeling/domain-model/entities/access-rules.md [hunk 1]
```diff
@@ -199,42 +199,68 @@ Non-persistable objects cannot have XPath constraints.
...
+From Studio Pro version 10.6.0 until version 10.16.0, the new access rule editor worked with normalized access rules. A normalized access rule is an access rule that has exactly one module role attached to it. See [Access Rule Normalization](#normalization), below, for the implications when you switch to the new editor for those versions.
 {{% /alert %}}
 
 ### Editor Layout
 
-Each module role from the module of the entity will have a default column for its access rights without an XPath. Next to this default column a module role can have more columns if XPath's have been added. The default column can not be deleted.
+The access rules editor displays each access rule belonging to the entity as an individual column. The header shows the modules role(s) the access rules apply to. When the module role name is long or there are multiple modules, the label is clipped. Hover over the header to show the full module role(s).
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/domain-model/entities/access-rules.md]: Should this be multiple ROLES?
```suggestion
The access rules editor displays each access rule belonging to the entity as an individual column. The header shows the modules role(s) the access rules apply to. When the module role name is long or there are multiple roles, the label is clipped. Hover over the header to show the full module role(s).
```

File: content/en/docs/refguide/modeling/domain-model/entities/access-rules.md [hunk 2]
```diff
@@ -199,42 +199,68 @@ Non-persistable objects cannot have XPath constraints.
...
 
-The rights in this column will only be granted if the XPath constraint is true.
+When there is a value shown in the **XPath constraint** row for a specific access rule, this means it has an XPath constraint. The access rule will only be granted if the XPath constraint is true. When an XPath constraint has a caption, this is shown instead of the XPath constraint itself.
 
-The new column will display the XPath constraint, or the **Caption** of the XPath constraint if one has been entered.
+### Entity Rights
+
+By setting the appropriate entity rights in an access rule, you can control the ability to create and/or delete objects of the entity. By default, an access rule has no entity rights. 
+
+When an access rule has entity rights, this is shown using the **Create** and/or **Delete** ({{% icon name="trash-can" %}}) icon in the entity rights row for the specific access rule.
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/domain-model/entities/access-rules.md]: Ideally need the icon for **Create**. Can't find it in the Mendix Icon Set. Do you have it as an svg?

**Review by MarkvanMents — COMMENTED** [review]
Partially proofread. Will finish first thing tomorrow.
See a couple of questions, below.

File: content/en/docs/refguide/modeling/domain-model/entities/access-rules.md (hunk 1 above)
Comment (brambrink) [reply content/en/docs/refguide/modeling/domain-model/entities/access-rules.md]: Yes, that was a mistake, I meant multiple module roles.

File: content/en/docs/refguide/modeling/domain-model/entities/access-rules.md [hunk 3]
```diff
@@ -259,84 +287,32 @@ The XPath constraint can be constructed easily using the **Path to user...** but
...
+You can also click **Read** or **Write** in the **Set all to** footer to enable or disable 'Read' or 'Read and Write' access for all attributes and associations in this column. 
 
 {{% alert color="info" %}}
-You cannot set *Write* access to attributes which are calculated. This includes attributes of type *Autonumber* and attributes where the attribute value is set to **Calculated**.
+You cannot set *Write* access to attributes which are calculated. This includes both attributes where the attribute value is set to **Calculated** and attributes of type *Autonumber*.
 {{% /alert %}}
 
-### Context menu {#context}
+Attributes and associations which are added to an entity in the domain model will default to allowing no access to any module roles. This improves security by forcing you to choose who can access any new information which is added to the domain model and is a change from the old editor where you could set default rights for new members (described in [Member Read and Write Rights Section](#member-access), above).
```
Comment (MarkvanMents) [inline content/en/docs/refguide/modeling/domain-model/entities/access-rules.md:310]: Is this new in 10.17 or has it always been in the new editor?

**Review by MarkvanMents — COMMENTED** [review]
I've finished my proofread. Please have a look and see if it all makes sense.
If it is good to go, I'll publish it alongside the rest of the 10.17 documentation when it is released later today.

File: content/en/docs/refguide/modeling/domain-model/entities/access-rules.md (hunk 3 above)
Comment (brambrink) [reply content/en/docs/refguide/modeling/domain-model/entities/access-rules.md:310]: This has been the case since 10.6.0, I moved this part from another place on the page. We are actually planning to add this functionality back in 10.18 based on customer feedback, but it is not there yet.

File: content/en/docs/community-tools/contribute-to-mendix-docs/icon-demo.md [hunk 4]
```diff
@@ -11,6 +11,10 @@ draft: true
 
 Mendix provides a standard set of approved icons in [The Mendix Icon Set](https://mendix.atlassian.net/l/cp/U89wu3oL). This icon set is downloaded to the [docs/static/mx-icons](https://github.com/mendix/docs/tree/development/static/mx-icons) library in SVG format for use in the docs via shortcodes, like this:
 
+```
+{{%/* icon name="three-dots-menu-horizontal" */%}}
```
Comment (brambrink) [inline content/en/docs/community-tools/contribute-to-mendix-docs/icon-demo.md:15]: Is this change intentional?

Comment (MarkvanMents) [reply content/en/docs/community-tools/contribute-to-mendix-docs/icon-demo.md:15]: Yes, - sorry to add it to your PR but I discovered that the instructions for adding and icon were incomplete so I added them when I worked out how to do it when I was putting the icons into your doc.

I can do it in another PR if you prefer.

File: content/en/docs/refguide/modeling/domain-model/entities/access-rules.md (hunk 3 above)
Comment (MarkvanMents) [reply content/en/docs/refguide/modeling/domain-model/entities/access-rules.md:310]: Great - I'll resolve this.

File: content/en/docs/community-tools/contribute-to-mendix-docs/icon-demo.md (hunk 4 above)
Comment (brambrink) [reply content/en/docs/community-tools/contribute-to-mendix-docs/icon-demo.md:15]: No worries, just wanted to ensure it wasn't included accidentally.

**Review by MarkvanMents — APPROVED** [review]
Approved. Ready to be published with the rest of the 10.17 documentation.
